### PR TITLE
update examples on blogs using the new sync APIs

### PR DESCRIPTION
**Pull request by db-vendia**

### CODE REVIEW

File: posts/2021-02-17-multiregion-serverless-apps.md [hunk 1]
```diff
@@ -172,11 +172,11 @@ Now, let's take a look at the user-visible resources in each node.  The `EwrStat
...
 
-![Weather Station Uni Page](https://d24nhiikxn5jns.cloudfront.net/optimized/user-images.githubusercontent.com..71095088140415440-5f4f6bfa-5181-496e-8a2b-5bcb06df93bb.png)
+![Weather Station Uni Page](https://user-images.githubusercontent.com/92179243/158901163-061b5389-0171-4626-a09f-077d7f94887c.jpg)
 
 ## Querying Data
 
-Recall that when we created our Uni, we seeded it with data from the `initial-state.json` file.  Let's go ahead and query that initial data, which is stored in every node in our uni.  First, click on the 'Open GraphQL Explorer' in the Vendia Share web application for the `EwrStation` node and list all of the Temperatures stored in the uni:
+Recall that when we created our Uni, we seeded it with data from the `initial-state.json` file.  Let's go ahead and query that initial data, which is stored in every node in our uni.  First, click on the 'Open GraphQL Explorer' in the Vendia Share web application for the `` node and list all of the Temperatures stored in the uni:
```
Comment (james-vendia) [inline posts/2021-02-17-multiregion-serverless-apps.md:179]:
```suggestion
Recall that when we created our Uni, we seeded it with data from the `initial-state.json` file.  Let's go ahead and query that initial data, which is stored in every node in our uni.  First, click on the 'Open GraphQL Explorer' in the Vendia Share web application for the `EwrStation` node and list all of the Temperatures stored in the uni:
```

Comment (james-vendia) [reply posts/2021-02-17-multiregion-serverless-apps.md:179]: Not sure why the node name was removed?

File: posts/2021-02-17-multiregion-serverless-apps.md [hunk 2]
```diff
@@ -216,17 +216,21 @@ So far we've assured ourselves that both nodes have a consistent view of the ini
...
 mutation addTemp {
- add_Temperature_async(
+ add_Temperature(
    input: {
      airportCode: "EWR",
      timestamp: "2021-01-01T03:00:00Z",
      temperature: 1.5
    }
- )
+   syncMode: NODE_LEDGERED){
```
Comment (james-vendia) [inline posts/2021-02-17-multiregion-serverless-apps.md:225]:
```suggestion
   syncMode: ASYNC){
```

Comment (james-vendia) [reply posts/2021-02-17-multiregion-serverless-apps.md:225]: Not sure if there's a reason to move to synch for this one.  Just minimizing the differences between old and new.

**Review by james-vendia — APPROVED** [review]
A few small changes included but not a blocker to merge before or on Monday.

Question - what was the intent to in moving some of the existing (async) mutations to NODE_LEDGERED vs ASYNC?  I think our recommended approach is still ASYNC, correct?

File: posts/2021-02-17-multiregion-serverless-apps.md (hunk 2 above)
Comment (db-vendia) [reply posts/2021-02-17-multiregion-serverless-apps.md]: I put them as `NODE_LEDGERED` out of laziness mostly. Most mutations had a `result` field that would result in a NULL value if the mutations was `ASYNC` so I left those as `NODE_LEDGERED` so the result would appear. I cleaned up the examples, and not they all should be `ASYNC`

**Pull request merged**